daemon/labels.go

```diff
@@ -232,16 +233,33 @@ func NewGetIdentityHandler(d *Daemon) GetIdentityHandler {
 }
 
 func (h *getIdentity) Handle(params GetIdentityParams) middleware.Responder {
-	lbls := labels.NewLabelsFromModel(params.Labels)
-	if id, err := LookupIdentityBySHA256(lbls.SHA256Sum()); err != nil {
-		if apierr, ok := err.(*apierror.APIError); ok {
-			return apierr
+	log.Debugf("GET /identity request: %+v", params)
+
+	identities := []*models.Identity{}
+	if params.Labels == nil {
+		outputList, err := kvstore.Client().ListPrefix(common.LabelIDKeyPath)
```

Comment:
Can you add a comment here? `// if labels is nil return all identities from the kvstore`